lnwallet/interface_test.go

```diff
@@ -2165,6 +2165,215 @@ func testChangeOutputSpendConfirmation(r *rpctest.Harness,
 	}
 }
 
+// testCreateSimpleTx checks that a call to CreateSimpleTx will return a
+// transaction that is equal to the one that is being created by SendOutputs in
+// a subsequent call.
+func testCreateSimpleTx(r *rpctest.Harness, w *lnwallet.LightningWallet,
+	_ *lnwallet.LightningWallet, t *testing.T) {
+
+	// We first mine a few blocks to ensure any transactions still in the
+	// mempool confirm.
+	_, err := r.Node.Generate(5)
```

Comment:
Would prefer if each test mined its own transactions once done.

Reply:
I don't remember why this was needed tbh. Fixed: https://github.com/lightningnetwork/lnd/pull/1228/commits/1287acde794dcac95bcb0ce54b270e8705387d90